README.md

```diff
@@ -48,13 +48,31 @@ export class HomeComponent {
     fromEvent(document, 'mousemove').subscribe()
   ];
 
-  // You can still use the opertator
+  // You can still use the operator
   ngOnInit() {
     interval(1000).pipe(untilDestroyed(this));
   }
 }
 ```
 
+You can set the `blackList` property if you **don't** want to unsubscribe from one or more subscriptions.
+
+```ts
+@UntilDestroy({ checkProperties: true, blackList: ['subject1$'] })
+@Component({})
+export class HomeComponent {
+  // subject$ will not be unsubscribed upon component destruction
+  subject1$: Subscription;
+  // subject2$ will be unsubscribed upon component destruction
+  subject2$: Subscription;
+
+  constructor() {
+    subject1$ = new Subject().subscribe();
```

Comment:
1. You need to add `this.subject$`
2. It's not a `subject$` - it's a subscription.